library/Fission/Random.hs

```diff
@@ -1,42 +1,32 @@
 -- | Random values
 module Fission.Random
-  ( byteString
-  , text
+  ( alphaNum
+  , alphaNumSymbol
   ) where
 
 import qualified Data.ByteString.Random as BS
-import           Data.Word8
 import qualified RIO.ByteString         as BS
+import qualified Fission.Internal.URL   as URL
+import           Data.Text              as T
+import           Data.Word8
+import           GHC.Natural
 
-import Fission.Internal.Bool (anyX)
 import Fission.Prelude
 
--- | Generate random 'Text'
-text :: Natural -> IO Text
-text amount = decodeUtf8Lenient <$> byteString amount
+-- | Generate random AlphaNumeric 'Text' with a given length
+alphaNum :: Natural -> IO Text
+alphaNum len = decodeUtf8Lenient . BS.filter isAsciiAlphaNum <$> bsRandomLength len
```

Comment:
While we're in here, let's get rid of the `IO` specialization

```haskell
alphaNum :: MonadIO m => Natural -> m Text
alphaNumSymbol :: MoandIO m => Natural -> m Text
```

And then we won't need to wrap this in `liftIO` at every call site

Reply:
@expede It didnt like this change 
![image](https://user-images.githubusercontent.com/1325608/70929903-49906280-1ff1-11ea-9b7f-6f336338bde7.png)
Any ideas why?

Reply:
You need to `liftIO`

Reply:
🤦‍♂